### ISSUE TITLE
Cannot sync aowow with world database

### ISSUE THREAD

**Issue by BeLove** [Sarjuuk/aowow#55]
Hi. Thanks for revival this project :)
I tried to install "from the clean" and confused at the last step.
My steps are:
1) Cloned repo
2) Successfully compiled MPQExtractor
3) Created sh script:

    ./MPQExtractor -e "DBFilesClient/*.*" -f -o out $1
    ./MPQExtractor -e "Interface/WorldMap/*.*" -f -o out $1
    ./MPQExtractor -e "Interface/FrameXML/GlobalStrings.lua" -f -o out $1
    ./MPQExtractor -e "Interface/TalentFrame/*.*" -f -o out $1
    ./MPQExtractor -e "Interface/Glues/Credits/*.*" -f -o out $1
    ./MPQExtractor -e "Interface/Icons/*.*" -f -o out $1
    ./MPQExtractor -e "Interface/Spellbook/*.*" -f -o out $1
    ./MPQExtractor -e "Interface/PaperDoll/*.*" -f -o out $1
    ./MPQExtractor -e "Interface/GLUES/CHARACTERCREATE/*.*" -f -o out $1
    ./MPQExtractor -e "Interface/Pictures/*.*" -f -o out $1
    ./MPQExtractor -e "Interface/PvPRankBadges/*.*" -f -o out $1
    ./MPQExtractor -e "Interface/FlavorImages/*.*" -f -o out $1
    ./MPQExtractor -e "Interface/Calendar/Holidays/*.*" -f -o out $1
    ./MPQExtractor -e "Interface/GLUES/LOADINGSCREENS/*.*" -f -o out $1

4) Moved this script to World of Warcraft/Data/ruRU directory (with MPQExtractor ofc)
5) Launched for next files in order:

    backup-ruRU.MPQ
    base-ruRU.MPQ
    expansion-locale-ruRU.MPQ
    expansion-speech-ruRU.MPQ
    lichking-locale-ruRU.MPQ
    lichking-speech-ruRU.MPQ
    locale-ruRU.MPQ
    patch-ruRU.MPQ
    patch-ruRU-2.MPQ
    patch-ruRU-3.MPQ
    speech-ruRU.MPQ

5) Moved new "out" (443 mb) dir to setup/mpqdata/ruRU
6) Created new user "aowow" with full access to aowow db (and select to chars, world and auth) + imported db_structure.sql file to aowow db
7) $ chmod 777 cache static config
8) php aowow --firstrun
Everything here is ok (setup db connections etc) and sync is ok. 

But most tables are empty. Except 	few like aowow_itemenchantment (2,656 rows), aowow_quests_startend (11,797 rows) and some others (just 4-5 tables). I tried to run
$ php aowow --sql
But it doesn't help. (log is ok, seems everything is fine, but tables are empty)

php aowow --sql
07:31:56          begin generation of achievementcategory, achievementcriteria, glyphproperties, itemenchantmentcondition, itemextendedcost, itemlimitcategory, itemrandomproppoints, lock, mailtemplate, scalingstatdistribution, scalingstatvalues, spellfocusobject, spellrange, spellvariables, totemcategory, talents, classes, factions, factiontemplate, holidays, icons, itemrandomenchant, races, shapeshiftforms, skillline, emotes, itemenchantment, achievement, creature, currencies, events, objects, pet, quests, quests_startend, spell, spelldifficulty, taxi, titles, items, spawns, zones, itemset, item_stats, source

07:31:56          SqlGen::generate() - copying achievement_category.dbc into aowow_achievementcategory

07:31:56          reading MPQdata from setup/mpqdata/ to list for first time use...
07:31:56          done

07:31:56           - reading and merging data from setup/mpqdata/ruRU/DBFilesClient/Achievement_Category.dbc
07:31:56  [OK]     - subscript 'achievementcategory' returned sucessfully
07:31:56          SqlGen::generate() - copying achievement_criteria.dbc into aowow_achievementcriteria
07:31:56           - reading and merging data from setup/mpqdata/ruRU/DBFilesClient/Achievement_Criteria.dbc
07:31:58  [OK]     - subscript 'achievementcriteria' returned sucessfully
07:31:58          SqlGen::generate() - copying glyphproperties.dbc into aowow_glyphproperties
07:31:58  [OK]     - subscript 'glyphproperties' returned sucessfully
07:31:58          SqlGen::generate() - copying spellitemenchantmentcondition.dbc into aowow_itemenchantmentcondition
07:31:58           - reading data from setup/mpqdata/ruRU/DBFilesClient/SpellItemEnchantmentCondition.dbc
07:31:58  [OK]     - subscript 'itemenchantmentcondition' returned sucessfully
07:31:58          SqlGen::generate() - copying itemextendedcost.dbc into aowow_itemextendedcost
07:31:58           - reading data from setup/mpqdata/ruRU/DBFilesClient/ItemExtendedCost.dbc
07:31:59  [OK]     - subscript 'itemextendedcost' returned sucessfully
07:31:59          SqlGen::generate() - copying itemlimitcategory.dbc into aowow_itemlimitcategory
07:31:59           - reading and merging data from setup/mpqdata/ruRU/DBFilesClient/ItemLimitCategory.dbc
07:31:59  [OK]     - subscript 'itemlimitcategory' returned sucessfully
07:31:59          SqlGen::generate() - copying randproppoints.dbc into aowow_itemrandomproppoints
07:31:59           - reading data from setup/mpqdata/ruRU/DBFilesClient/RandPropPoints.dbc
07:31:59  [OK]     - subscript 'itemrandomproppoints' returned sucessfully
07:31:59          SqlGen::generate() - copying lock.dbc into aowow_lock
07:31:59  [OK]     - subscript 'lock' returned sucessfully
07:31:59          SqlGen::generate() - copying mailtemplate.dbc into aowow_mailtemplate
07:31:59           - reading and merging data from setup/mpqdata/ruRU/DBFilesClient/MailTemplate.dbc
07:31:59  [OK]     - subscript 'mailtemplate' returned sucessfully
07:31:59          SqlGen::generate() - copying scalingstatdistribution.dbc into aowow_scalingstatdistribution
07:31:59  [OK]     - subscript 'scalingstatdistribution' returned sucessfully
07:31:59          SqlGen::generate() - copying scalingstatvalues.dbc into aowow_scalingstatvalues
07:31:59  [OK]     - subscript 'scalingstatvalues' returned sucessfully
07:31:59          SqlGen::generate() - copying spellfocusobject.dbc into aowow_spellfocusobject
07:31:59           - reading and merging data from setup/mpqdata/ruRU/DBFilesClient/SpellFocusObject.dbc
07:31:59  [OK]     - subscript 'spellfocusobject' returned sucessfully
07:31:59          SqlGen::generate() - copying spellrange.dbc into aowow_spellrange
07:31:59           - reading and merging data from setup/mpqdata/ruRU/DBFilesClient/SpellRange.dbc
07:31:59  [OK]     - subscript 'spellrange' returned sucessfully
07:31:59          SqlGen::generate() - copying spelldescriptionvariables.dbc into aowow_spellvariables
07:31:59           - reading data from setup/mpqdata/ruRU/DBFilesClient/SpellDescriptionVariables.dbc
07:31:59  [OK]     - subscript 'spellvariables' returned sucessfully
07:31:59          SqlGen::generate() - copying totemcategory.dbc into aowow_totemcategory
07:31:59           - reading and merging data from setup/mpqdata/ruRU/DBFilesClient/TotemCategory.dbc
07:31:59  [OK]     - subscript 'totemcategory' returned sucessfully
07:31:59          SqlGen::generate() - filling aowow_talents with data
07:31:59  [OK]     - subscript 'talents' returned sucessfully
07:31:59          SqlGen::generate() - filling aowow_classes with data
07:31:59           - reading and merging data from setup/mpqdata/ruRU/DBFilesClient/Spell.dbc

Any ideas?
Thanks!

**Comment by BeLove** [comment]
Ok after some magic database partially synchronized.
But no icons and some tables are still empty.
If it's ok - I can post link to db here.

**Comment by BeLove** [comment]
After few reinstallation now I have a mostly working db (except few issues), ticket can be closed.
Probably for someone provided info in my first message can help in future.

**Comment by Sarjuuk** [comment]
Sorry for the delay. If you still have any problems, feel free to ask me.

Also, it would be nice if you could describe how you solved this error here. Maybe i can easily iron it out or at least anticipate it in the future.

**Comment by BeLove** [comment]
Actually I don't know, I just tried few times

    $ php aowow --sql
    $ php aowow --firstrun

And now db looks ok. But I have another issue - not all icons and images are extracted from MPQ files. I tried to extract them from all MPQ files that I have found. And solution for me - take missing icons from another similar project - https://github.com/Exodius/WoW-Armory/ (only big icons, anothers sizes are different from which we need for AoWoW) and convert tiny and small icons with imagemagick (from big icons).
